scss/_pagination.scss

```diff
@@ -5,7 +5,20 @@
 // > BREADCRUMB
 //
 .breadcrumb {
+
+  width: $breadcrumb-width;
+  font-size: $breadcrumb-font-size;
   background-color: $breadcrumb-bg;
+
+
+  &::before {
+    float: left; // Suppress inline spacings and underlining of the separator
+    padding-right: $breadcrumb-item-padding;
+    // stylelint-disable-next-line
+    font-family: "Material Icons Sharp";
+    color: $breadcrumb-divider-color;
+    content: "\e5cc";
```

Comment:
We should follow the pattern of
https://github.com/az-digital/arizona-bootstrap/pull/303
(with a name rather than a codepoint, for example).